2024-08-10 21:08:24,408 - INFO - Adding job tentatively -- it will be properly scheduled when the scheduler starts
2024-08-10 21:08:24,408 - INFO - Added job "run_procedure_sequence" to job store "default"
2024-08-10 21:08:24,408 - INFO - Scheduler started
2024-08-10 21:08:26,854 - INFO - Adding job tentatively -- it will be properly scheduled when the scheduler starts
2024-08-10 21:08:26,854 - INFO - Added job "run_procedure_sequence" to job store "default"
2024-08-10 21:08:26,854 - INFO - Scheduler started
2024-08-10 21:08:26,872 - INFO - Watching for file changes with StatReloader
2024-08-10 21:08:28,587 - WARNING - Not Found: /images/favicon.ico
2024-08-10 21:08:37,230 - WARNING - Not Found: /ml_models/ml-report/images/favicon.ico
2024-08-10 21:11:00,009 - INFO - Running job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-10 21:11:00 BST)" (scheduled at 2024-08-10 21:11:00+01:00)
2024-08-10 21:11:00,009 - INFO - Running job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-10 21:11:00 BST)" (scheduled at 2024-08-10 21:11:00+01:00)
2024-08-10 21:11:00,150 - INFO - The input start time: 2024-08-10 20:00:00
2024-08-10 21:11:00,151 - INFO - The input start time: 2024-08-10 20:00:00
2024-08-10 21:11:00,761 - ERROR - Job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-10 21:16:00 BST)" raised an exception
Traceback (most recent call last):
  File "C:\Users\sunny\anaconda3\envs\myenv311test\Lib\site-packages\apscheduler\executors\base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\updater.py", line 73, in run_procedure_sequence
    run_IG(ticker)
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\IG_session\run_IG.py", line 165, in run_IG
    ig_service.create_session()
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\IG_session\ig_service.py", line 393, in create_session
    self._set_headers(response.headers, True)
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\IG_session\ig_service.py", line 450, in _set_headers
    self.CLIENT_TOKEN = response_headers['CST']
                        ~~~~~~~~~~~~~~~~^^^^^^^
  File "C:\Users\sunny\anaconda3\envs\myenv311test\Lib\site-packages\requests\structures.py", line 52, in __getitem__
    return self._store[key.lower()][1]
           ~~~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'cst'
2024-08-10 21:11:16,341 - INFO - Job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-10 21:16:00 BST)" executed successfully
2024-08-10 21:16:00,002 - INFO - Running job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-10 21:21:00 BST)" (scheduled at 2024-08-10 21:16:00+01:00)
2024-08-10 21:16:00,002 - INFO - Running job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-10 21:21:00 BST)" (scheduled at 2024-08-10 21:16:00+01:00)
2024-08-10 21:16:00,007 - INFO - The input start time: 2024-08-10 20:00:00
2024-08-10 21:16:00,007 - INFO - The input start time: 2024-08-10 20:00:00
2024-08-10 21:16:18,406 - INFO - Job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-10 21:21:00 BST)" executed successfully
2024-08-10 21:16:18,788 - INFO - Job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-10 21:21:00 BST)" executed successfully
2024-08-11 07:28:30,162 - INFO - Adding job tentatively -- it will be properly scheduled when the scheduler starts
2024-08-11 07:28:30,162 - INFO - Added job "run_procedure_sequence" to job store "default"
2024-08-11 07:28:30,162 - INFO - Scheduler started
2024-08-11 07:28:32,679 - INFO - Adding job tentatively -- it will be properly scheduled when the scheduler starts
2024-08-11 07:28:32,679 - INFO - Added job "run_procedure_sequence" to job store "default"
2024-08-11 07:28:32,679 - INFO - Scheduler started
2024-08-11 07:28:32,697 - INFO - Watching for file changes with StatReloader
2024-08-11 07:31:00,013 - INFO - Running job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-11 07:31:00 BST)" (scheduled at 2024-08-11 07:31:00+01:00)
2024-08-11 07:31:00,013 - INFO - Running job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-11 07:36:00 BST)" (scheduled at 2024-08-11 07:31:00+01:00)
2024-08-11 07:31:00,237 - INFO - The input start time: 2024-08-11 06:00:00
2024-08-11 07:31:00,241 - INFO - The input start time: 2024-08-11 06:00:00
2024-08-11 07:31:01,003 - ERROR - Job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-11 07:36:00 BST)" raised an exception
Traceback (most recent call last):
  File "C:\Users\sunny\anaconda3\envs\myenv311test\Lib\site-packages\apscheduler\executors\base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\updater.py", line 73, in run_procedure_sequence
    run_IG(ticker)
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\IG_session\run_IG.py", line 165, in run_IG
    ig_service.create_session()
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\IG_session\ig_service.py", line 393, in create_session
    self._set_headers(response.headers, True)
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\IG_session\ig_service.py", line 450, in _set_headers
    self.CLIENT_TOKEN = response_headers['CST']
                        ~~~~~~~~~~~~~~~~^^^^^^^
  File "C:\Users\sunny\anaconda3\envs\myenv311test\Lib\site-packages\requests\structures.py", line 52, in __getitem__
    return self._store[key.lower()][1]
           ~~~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'cst'
2024-08-11 07:31:01,004 - ERROR - Job "run_procedure_sequence (trigger: interval[0:05:00], next run at: 2024-08-11 07:36:00 BST)" raised an exception
Traceback (most recent call last):
  File "C:\Users\sunny\anaconda3\envs\myenv311test\Lib\site-packages\apscheduler\executors\base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\updater.py", line 73, in run_procedure_sequence
    run_IG(ticker)
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\IG_session\run_IG.py", line 165, in run_IG
    ig_service.create_session()
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\IG_session\ig_service.py", line 393, in create_session
    self._set_headers(response.headers, True)
  File "C:\Users\sunny\Desktop\Development\python\TradePro\prices\IG_session\ig_service.py", line 450, in _set_headers
    self.CLIENT_TOKEN = response_headers['CST']
                        ~~~~~~~~~~~~~~~~^^^^^^^
  File "C:\Users\sunny\anaconda3\envs\myenv311test\Lib\site-packages\requests\structures.py", line 52, in __getitem__
    return self._store[key.lower()][1]
           ~~~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'cst'
2024-08-11 07:32:32,840 - INFO - C:\Users\sunny\Desktop\Development\python\TradePro\ml_models\utils\analysis_comments.py changed, reloading.
2024-08-11 07:32:36,458 - INFO - Adding job tentatively -- it will be properly scheduled when the scheduler starts
2024-08-11 07:32:36,458 - INFO - Added job "run_procedure_sequence" to job store "default"
2024-08-11 07:32:36,459 - INFO - Scheduler started
2024-08-11 07:32:36,480 - INFO - Watching for file changes with StatReloader
